examples/tls-example.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+import binascii
+from iroha import IrohaCrypto
+from iroha import Iroha, IrohaGrpc
+import sys
+
+if sys.version_info[0] < 3:
+    raise Exception('Python 3 or a more recent version is required.')
+
+print("""
+
+This example works only through a TLS connection, you must first configure torii_tls_params
+https://iroha.readthedocs.io/en/master/configure/torii-tls.html.
```

Comment:
wrong link, it is not working:
```suggestion
https://iroha.readthedocs.io/en/main/configure/torii-tls.html
```